tests/phpunit/tests/functions.php

```diff
@@ -1917,7 +1917,7 @@ public function data_validate_file() {
 	 */
 	public function test_wp_is_stream( $path, $expected ) {
 		if ( ! extension_loaded( 'openssl' ) && false !== strpos( $path, 'https://' ) ) {
-			$this->markTestSkipped( 'The openssl PHP extension is not loaded.' );
+			$this->fail( 'The openssl PHP extension is not loaded.' );
```

Comment:
Same thing again. Failing tests should only be done when the code under test doesn't run correctly. Not when the environment in which the tests are run does not comply with the desired environment.